src/ui/component/viewers/audioViewer.jsx

```diff
@@ -1,5 +1,3 @@
-// @flow
```

Comment:
I removed flow from this file because there were ~30 type errors. If we have flow in a file, it should cover everything so we don't need to constantly use `--no-verify`